_rules/button-non-empty-accessible-name-97a4e1.md

```diff
@@ -42,6 +40,8 @@ Each target element has an [accessible name][] that is not empty (`""`).
 
 ## Background
 
+Some elements have a role of `button` and a default accessible name defined by the [HTML Accessibility API Mapping][html aam input button], for example `input` elements whose `type` [attribute value][] is either `submit` or `reset`. This rule considers that these default names can be descriptive and therefore does not fail them.
```

Comment:
Only thought is that this statement could still be moved into the assumption, i.e., we assume that if the default name is used then it is intended and descriptive for the situation.